### PR TITLE
Add a full_title helper

### PR DESCRIPTION
**Pull request by hungkieu**
ready

### CODE REVIEW

**Comment by hungkieu** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield :title %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title yield :title %></title>
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    "application", media: "all",
                                               "data-turbolinks-track": "reload" %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: chú ý ident

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<% provide :title, "About" %>
+<% provide :title, t("static_pages.about") %>
 <h1>About</h1>
```
Comment (thangtx88) [inline app/views/static_pages/about.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -1,9 +1,7 @@
...
-<h1>Help</h1>
+<% provide :title, t("static_pages.home") %>
+<h1>Sample App</h1>
 <p>
-  Get help on the Ruby on Rails Tutorial at the
-  <a href="http://www.railstutorial.org/help">Rails Tutorial help section</a>.
-  To get help on this sample app, see the
-  <a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
-  book</a>.
+  This is the home page for the
```
Comment (thangtx88) [inline app/views/static_pages/home.html.erb]: I18n

**Comment by thangtx88** [comment]
commented

**Comment by hungkieu** [comment]
ready

**Comment by hungkieu** [comment]
ready

**Comment by hungkieu** [comment]
ready

**Comment by hungkieu** [comment]
ready

**Comment by hungkieu** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**